src/ApiService/Tests/OrmTest.cs

```diff
@@ -276,5 +276,25 @@ public void TestEventSerialization2()
             var actualEvent = converter.ToRecord<EventMessage>(te);
             Assert.Equal(expectedEvent, actualEvent);
         }
+
+        record Entity3(
+            [PartitionKey] int Id,
+            [RowKey] string TheName,
+            Container Container
+        ) : EntityBase();
+
+        [Fact]
+        public void TestContainerSerialization()
```

Comment:
This Test is probably going to work even without your change. 
The problem is the  default serialization will translate a container to an object 
``` json 
{
   "ContainerName": "<name>"
}
```

but in existing object a container is represented by a simple string 

This test make sure that we can deserialize what **WE** serialize but it does not show that w can deserialize existing objects